boto3/s3/inject.py

```diff
@@ -91,6 +94,27 @@ def download_file(self, Bucket, Key, Filename, ExtraArgs=None,
         extra_args=ExtraArgs, callback=Callback)
 
 
+def copy_file(self, SrcBucket, SrcKey, DestBucket, DestKey,
+              ExtraArgs=None, Callback=None, Config=None):
+    """Copy one S3 object to another s3 object.
+
+    Usage::
+
+        import boto3
+        s3 = boto3.resource('s3')
+        s3.meta.client.copy_file('bucket1', 'hello.txt', 'bucket2', 'hello.txt')
```

Comment:
Hi its nice for bucket to bucket transfer. I would like move the file completely from folder 'A' in bucket to different folder 'B' in same bucket. Means i want archived the file from one folder to another folder in same bucket. Could you please let us know, how can we able to do.

Thanks & regards,
Srinivasa Reddy B
PhNo: +91-8105856509